cmd/kubeadm/app/util/version.go

```diff
@@ -61,8 +65,7 @@ var (
 //  latest-1    (latest release in 1.x, including alpha/beta)
 //  latest-1.0  (and similarly 1.1, 1.2, 1.3, ...)
 func KubernetesReleaseVersion(version string) (string, error) {
-	ver := normalizedBuildVersion(version)
-	if len(ver) != 0 {
+	if ver, err := normalizeBuildVersion(version); err == nil {
```

Comment:
I think this we can simplify  normalize -> split -> normalize into split -> normalize only. 
Wdyt?

Reply:
@fabriziopandini I agree. I guess `normalizeBuildVersion` can be simplified even further to return `bool` instead of `error` as errors are ignored anyway.